components/footer/FooterMenu.tsx

```diff
@@ -12,7 +12,11 @@ const menuStyle = {
   justifyContent: "center",
   maxWidth: 606,
   maxHeight: 44,
-  flex: 1
+  flex: 1,
+  paddingX:{
```

Comment:
```suggestion
  paddingX: {
```
can we please modify eslint to add space after the object key?